packages/atomic/src/components/insight/atomic-insight-full-search-button/atomic-insight-full-search-button.pcss

```diff
@@ -0,0 +1,28 @@
+@import '../../../global/global.pcss';
+
+:host {
+  &::part(full-search-button) {
+    background-color: #949aa2;
+    height: 27px;
+    width: 27.75px;
+    clip-path: polygon(0 0, 100% 100%, 100% 0);
+    border-radius: 0;
+    border: none;
+    position: absolute;
```

Comment:
The fact that this is absolutely positioned means that there's only one place where it can be rendered, and you cannot really move it around to place it differently in the interface.

I would propose to not have that kind of precise hardcoded styles directly on the component itself, and to instead use non-component section to do that kinda thing.

Another way to go about this could be to have a new slot on the insight-interface to place the button.

Reply:
what did you have in mind exactly for the slot? 🤔